packages/app/src/app/pages/Sandbox/Editor/Workspace/items/Deployment/Netlify.js

```diff
@@ -77,6 +92,43 @@ class NetlifyDeployment extends Component {
                   <Deploy key={deployment.netlifySite.uid}>
                     <Name light>{deployment.netlifySite.name}</Name>
                     {!deployment.building && <div>Building</div>}
+                    {functions.length ? (
```

Comment:
Nice! Maybe we could also add header "Actions" for the buttons under the functions to make sure that people don't confuse those buttons with the functions.

Reply:
Done :D